test/src/main/java/org/axonframework/test/aggregate/AggregateTestFixture.java

```diff
@@ -345,15 +359,10 @@ public ResultValidator andThenTimeAdvancesTo(Instant newDateTime) {
      * @param aggregateDescriptor A {@link ScopeDescriptor} describing the aggregate under test
      * @param deadlineMessage     The {@link DeadlineMessage} to be handled
      */
-    protected void handleDeadline(ScopeDescriptor aggregateDescriptor, DeadlineMessage<?> deadlineMessage) {
+    protected void handleDeadline(ScopeDescriptor aggregateDescriptor, DeadlineMessage<?> deadlineMessage)
+            throws Exception {
         ensureRepositoryConfiguration();
-        DefaultUnitOfWork.startAndGet(deadlineMessage).execute(() -> {
```

Comment:
I don't fully comprehend why this call not longer needs to be a part of a `UnitOfWork`.
Do you mind elaborating on this Milan?

Reply:
It is moved to the `StubDeadlineManager`. Take a look at `#consumeDeadline` method.

Reply:
Aaah, thanks for pointing that out! :D